DESCRIPTION

```diff
@@ -32,7 +32,8 @@ Imports:
     purrr,
     tibble,
     dplyr,
-    rstudioapi
+    rstudioapi,
+    stringr
```

Comment:
We removed `stringr` as a dependency because it depends on `stringi` that can be hard to install, especially from source, so I'd prefer your refactoring your code without `stringr`.